src/fragments/lib/utilities/js/hub.mdx

```diff
@@ -73,20 +86,19 @@ The `event` field is recommended to be a small string without spaces such as `si
 
 ### Stop Listening
 
-Hub provides a way to stop listening for messages with `Hub.remove(channel: string | RegExp, listener: callback)`. This may be useful if you no longer need to receive messages in your application flow, as well as to avoid any memory leaks on low powered devices when you are sending large amounts of data through Hub on multiple channels.
+Hub provides a way to stop listening for messages with calling the result of the `.listen` function. This may be useful if you no longer need to receive messages in your application flow, as well as to avoid any memory leaks on low powered devices when you are sending large amounts of data through Hub on multiple channels.
 
-### Channels
-A channel is a logical group name that you use to organize messages and listen on. These are strings and completely up to you as the developer to define for dispatching or listening. However, while you can dispatch to any channel, ***Amplify protects certain channels*** and will flag a warning as sending unexpected payloads could have undesirable side effects (such as impacting authentication flows). The protected channels are currently:
+To stop listening to a certain event, you need to wrap the listener function to a variable and call it once you no longer need it:
+```javascript
 
-* core
-* auth
-* api
-* analytics
-* interactions
-* pubsub
-* storage
-* xr
-* datastore
+/* start listening for messages */
+const hubEventListener = Hub.listen(/.*/, (data) => {
```

Comment:
I think `cancelToken` might be a better variable name.

Reply:
I feel like `cancelToken` is a little confusing for me. Can you help me understand a bit why that would be better?

Reply:
@haverchuck were you able to discuss it with @abdallahshaban557?

Reply:
How about `hubListenerCancelToken`?